code/modules/spells/changeling/regenerative_stasis.dm

```diff
@@ -67,7 +67,7 @@
 	C.update_canmove()
 	if(M_HUSK in C.mutations) //Yes you can regenerate from being husked if you played dead beforehand, but unless you find a new body, you can not regenerate again.
 		to_chat(C, "<span class='notice'>This host body has become corrupted, either through a mishap, or betrayal by a member of the hivemind. We must find a new form, lest we lose ourselves to the void and become dust.</span>")
-		if(C.dna in changeling.absorbed_dna)
+		if(C.dna in changeling?.absorbed_dna)
```

Comment:
is this atomic? why tho

Reply:
This prevents a runtime if regenerate is cast by a non-changeling, which normally can't happen
Yes, it is atomic, the end of the vault has a wish granter and one wish you can ask for is immortality. This PR unsnowflakes that by just making it a subtype of the ling regeneration.